### PR TITLE
Bug/cuke profile updates

### PR DESCRIPTION
**Pull request by Soujanya2107**
Verifies successful updates

### CODE REVIEW

**Comment by radek1st** [comment]
Thanks, also please check the updated value is now visible on the profile page after a reload.

File: tests/javascript-cucumber/Features/step_definitions/steps.js [hunk 1]
```diff
@@ -307,7 +308,13 @@ module.exports = function () {
         await this.driver.wait(until.elementIsVisible(buttonelem), 60 * 1000);
         buttonelem.click();
     });
-
+    
+    this.Then(/^user is displayed with email updated message$/, async function () {
+        let elem = await this.driver.wait(until.elementLocated(By.className('ant-alert-description')), allowedTimeout);
+        let response = await elem.getText()
+        assert(response === "Email updated. Please check your new email to verify it.")
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/step_definitions/steps.js:315]: that's interesting, I didnt realise we do that. Maybe you can also add a check to see if email arrives, click to validate, and try to log in with new email?

Comment (radek1st) [reply tests/javascript-cucumber/Features/step_definitions/steps.js:315]: you can see how I did this yesterday in the signup cukes

**Comment by Soujanya2107** [comment]
Yeah I am on it Radek , unfortunately my internet is too bad today , tests
are not getting through, not even a step unfortunately.
Finding a way out.sorry for the delay caused

**Comment by Soujanya2107** [comment]
> Thanks, also please check the updated mobile is now visible on the profile page after a reload. --This is added.

File: tests/javascript-cucumber/Features/profile_update.feature [hunk 2]
```diff
@@ -6,20 +6,21 @@ Feature: User Profile Update
...
 
   Scenario: User updates email
 
     Given user logs in to their validated account
     And navigates to My profile page
     And user fills in the new email address and the password in the form
     When user clicks on change email button
-    Then user is displayed with success message
+    Then user is displayed with email updated message
+    Then refreshes the page and verifies if email is updated
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/profile_update.feature:18]: Change to And. There should be only one Given, When, Then

File: tests/javascript-cucumber/Features/profile_update.feature [hunk 3]
```diff
@@ -6,20 +6,21 @@ Feature: User Profile Update
...
 
   Scenario: User updates the phone number
 
     Given user logs in to their validated account
     And navigates to My profile page
     When user fills in the new phone number in the form
     And user clicks on submit changes button
-    Then user is displayed with alert message
+    Then user is displayed with success message,refreshed and verified
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/profile_update.feature:26]: Split to two steps:
1) Then user is displayed with success message
2) And updated phone is visible on page when refreshed

File: tests/javascript-cucumber/Features/step_definitions/steps.js [hunk 4]
```diff
@@ -16,9 +16,11 @@ let generateRandomString = function() {
     return "rp" + Math.floor(Math.random() * 10000000000000);
 }
 let randomEmail = generateRandomString() + "@apiemail.org";
-let randomPassword = generateRandomString() + "Nsw$5%9Yplr";
+let randomPassword = "Nsw$5%9Yplr";
+let randEmail = generateRandomString() + "@apiemail.org";
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/step_definitions/steps.js:20]: why do we have `randEmail` and `randomEmail`?

Comment (Soujanya2107) [reply tests/javascript-cucumber/Features/step_definitions/steps.js:20]: with the same variable it broke the SIGNUP cuke test and few more, having a seperate variable for signup fixed it.

**Pull request closed without merging**